x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -104,7 +104,9 @@ export function WorkspacePanelWrapper({
         <EuiScreenReaderOnly>
           <h1 data-test-subj="lns_ChartTitle">
             {title ||
-              i18n.translate('xpack.lens.chartTitle.unsaved', { defaultMessage: 'Unsaved' })}
```

Comment:
This is already part of japanese and korean translation files, it would make sense to remove it from there so the translation matches.